src/components/PopoverWithMeasuredContent.js

```diff
@@ -1,19 +1,21 @@
 import _ from 'underscore';
-import React, {Component} from 'react';
+import React, {useState, useEffect} from 'react';
 import PropTypes from 'prop-types';
 import {View} from 'react-native';
 import lodashGet from 'lodash/get';
 import Popover from './Popover';
 import {propTypes as popoverPropTypes, defaultProps as defaultPopoverProps} from './Popover/popoverPropTypes';
-import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+import useWindowDimensions from '../hooks/useWindowDimensions';
+import {windowDimensionsPropTypes} from './withWindowDimensions';
 import CONST from '../CONST';
 import styles from '../styles/styles';
 import {computeHorizontalShift, computeVerticalShift} from '../styles/getPopoverWithMeasuredContentStyles';
 
 const propTypes = {
     // All popover props except:
     // 1) anchorPosition (which is overridden for this component)
-    ..._.omit(popoverPropTypes, ['anchorPosition']),
+    // 2) windowDimensionsPropTypes (which are replaced by useWindowDimensions)
+    ..._.omit(popoverPropTypes, ['anchorPosition', ..._.keys(windowDimensionsPropTypes)]),
```

Comment:
Why are we omitting `windowDimensionsPropTypes` propTypes?

Reply:
@luacmartins @s77rt we could either leave withWindowDimensions hoc or change to useWindowDimensions hook. propTypes for this components uses Popover proptypes which uses Modal proptypes which use withWindowDimensions and isn't rewritten to functional component yet.
If this decision looks strange we could just leave withWindowDimensions as is.

Reply:
Also I think we should maintain consistency for all related components so if we changing them all to hooks it's better to use such a transition with props omission so when Modal will be rewritten nothing will be broken.

Reply:
First of all it's not right to put ...windowDimensionsPropTypes here https://github.com/Expensify/App/blob/df2e3eb65b81216aa8d29c8738bd16969299d161/src/components/Modal/modalPropTypes.js#L69 in modalPropTypes and then use it everywhere.

Reply:
Just wanted to get rid of hoc as we use hook, but it seems to be useful